public/pages/utils/helpers.js

```diff
@@ -74,14 +74,19 @@ export const appendCommentsAction = (columns, httpClient) => {
 
 export async function getIsAgentConfigured(dataSourceId){
   const assistantClient = getAssistantClient();
-  const res = await assistantClient.agentConfigExists(
-    [
-      SUMMARY_AGENT_CONFIG_ID,
-      LOG_PATTERN_SUMMARY_AGENT_CONFIG_ID,
-    ],
-    { dataSourceId: dataSourceId }
-  );
-  return res.exists;
+  try{
+    const res = await assistantClient.agentConfigExists(
+      [
+        SUMMARY_AGENT_CONFIG_ID,
+        LOG_PATTERN_SUMMARY_AGENT_CONFIG_ID,
+      ],
+      { dataSourceId: dataSourceId }
+    );
+    return res.exists;
+  }
+  catch(e){
+    return false;
```

Comment:
Minor: If the call fails, we may need to notify the error message. Otherwise, we won't know whether it's dependency error or default value because of absence of plugin